vcpkg.json

```diff
@@ -2,19 +2,20 @@
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
   "name": "chatterino",
   "version": "2.0.0",
-  "dependencies": [
-    "benchmark",
-    "boost-asio",
-    "boost-signals2",
-    "boost-foreach",
-    "boost-interprocess",
-    "boost-random",
-    "boost-variant",
-    "gtest",
-    "openssl",
-    "qt5-multimedia",
-    "qt5-tools"
-  ],
+    "dependencies": [
+        "benchmark",
+        "boost-asio",
+        "boost-signals2",
+        "boost-foreach",
+        "boost-interprocess",
+        "boost-random",
+        "boost-variant",
+        "boost-circular-buffer",
+        "gtest",
+        "openssl",
+        "qt5-multimedia",
+        "qt5-tools"
+    ],
```

Comment:
Adding the `boost-circular-buffer` entry is fine, even if it's unrelated to the PR since it's a small change, but could you reformat this change so it only sees the single `boost-circular-buffer` line change?